crawler/file_handler.py

```
@@ -73,20 +79,150 @@ def _extract_pdf(path: Path) -> tuple[str, str]:
+    current_cell: tuple[int, int] | None = None
+    cell_data: dict[tuple[int, int], list[str]] = {}
+
+    def flush_table() -> str:
+        if not cell_data:
+            return ""
+        # pyhwp는 colspan/rowspan으로 가려진 ghost 셀을 저장하지 않는다.
+        # 실제 LIST_HEADER 엔트리가 있는 셀만 row별로 모아 출력하면
+        # 빈 "|  |" 가 생기지 않는다.
+        rows_dict: dict[int, list[tuple[int, str]]] = defaultdict(list)
+        for (r, c), lines in cell_data.items():
+            text = " / ".join(l for l in lines if l.strip())
```

Ruff (0.15.12)

[error] Ambiguous variable name: `l`

(E741)